src/runtime_src/core/edge/common_em/config.h

```diff
@@ -251,6 +251,7 @@ namespace xclemulation{
   std::string getRunDirectory();
   
   std::map<std::string,std::string> getEnvironmentByReadingIni();
+  int getIPName2Index(const char *name,const char* ipLayoutbuf);
```

Comment:
Uncommented headers! :-(

@stsoe is there a coding guide for the team?

Reply:
added comments to the headers